README_INFRAESTRUCTURE.md

```diff
@@ -0,0 +1,789 @@
+      "ejector": "<ejector_address>",
+      "deployer": "<deployer_address>",
+      "initalPausedStatus": 0
+    },
+    "minimumStakes": [],  
+    "strategyWeights": [],
+    "operatorSetParams": [],
+    "uri": ""
+  }
+```
+
+You can find an example config file in `contracts/script/deploy/config/holesky/aligned.holesky.config.json`.
+
+To upgrade the Service Manager Contract in Testnet/Mainnet, run:
+
+```bash
+make upgrade_aligned_contracts
+```
+
+To upgrade the Registry Coordinator in Testnet/Mainnet, run:
+
+```bash
+make upgrade_registry_coordinator
+```
+
+Make sure to set environment variables in a `.env` file in the same directory as the upgrade script (`contracts/scripts/`).
+
+### Bindings
+
+Also make sure to re-generate the Go smart contract bindings:
+
+```bash
+make bindings
+```
+
+### Deployment
+
+To build go binaries run:
+
+```bash
+make build_binaries
+```
+
+## Metrics
+
+### Aggregator Metrics
+
+Aggregator metrics are exposed on the `/metrics` endpoint.
+
+If you are using the default config, you can access the metrics on `http://localhost:9091/metrics`.
+
+To run Prometheus and Grafana just run:
+
+```bash
+make run_metrics
+```
+
+Then you can access Grafana on `http://localhost:3000` with the default credentials `admin:admin`.
+
+If you want to install Prometheus and Grafana manually, you can follow the instructions below.
+
+To install Prometheus, you can follow the instructions on the [official website](https://prometheus.io/docs/prometheus/latest/getting_started/).
+
+To install Grafana, you can follow the instructions on the [official website](https://grafana.com/docs/grafana/latest/setup-grafana/installation/).
+
+## Explorer
+
+### Minimum Requirements
+
+- [Erlang 26](https://github.com/asdf-vm/asdf-erlang)
+- [Elixir 1.16.2](https://elixir-ko.github.io/install.html), compiled with OTP 26
+- [Docker](https://docs.docker.com/get-docker/)
+
+### DB Setup
+
+To setup the explorer, an installation of the DB is needed.
+
+First you'll need to install docker if you don't have it already. You can follow the instructions [here](https://docs.docker.com/get-docker/).
+
+The explorer uses a PostgreSQL database. To build and start the DB using docker, just run:
+
+```bash
+make build_db
+```
+
+This will build the docker image to be used as our database.
+
+After this, both `make run_explorer` and `make run_devnet_explorer` (see [this](#running-for-local-devnet) for more details) will automatically start, setup and connect to the database, which will be available on `localhost:5432` and the data is persisted in a volume.
+
+<details>
+
+<summary>
+  (Optional) The steps to manually execute the database are as follows...
+</summary>
+
+- Run the database container, opening port `5432`:
+
+```bash
+make run_db
+```
+
+- Configure the database with ecto running `ecto.create` and `ecto.migrate`:
+
+```bash
+make ecto_setup_db
+```
+
+- Start the explorer:
+
+```bash
+make run_explorer # or make run_devnet_explorer
+```
```

Comment:
I would clarify that after make build_db, doing only make run_explorer is enough to boot up the explorer
```suggestion
- Start the explorer (this will also `run_db` and `ecto_setup_db`):

```bash
make run_explorer # or make run_devnet_explorer
```
```